zenoh/src/net/routing/interceptor/access_control.rs

```diff
@@ -235,6 +235,23 @@ impl InterceptorTrait for IngressAclEnforcer {
             .or_else(|| ctx.full_expr());
 
         match &ctx.msg.body {
+            NetworkBody::Request(Request {
+                payload: RequestBody::Query(_),
+                ..
+            }) => {
+                if self.action(AclMessage::Query, "Query (ingress)", key_expr?) == Permission::Deny
+                {
+                    return None;
+                }
+            }
+            NetworkBody::Response(Response { wire_expr, .. }) => {
+                // @TODO: Remove wire_expr usage when issue #1255 is implemented
+                if self.action(AclMessage::Reply, "Reply (ingress)", wire_expr.as_str())
+                    == Permission::Deny
```

Comment:
This code, and its equivalent egress version, will need to be updated once #1255 is implemented.